x/auth/client/cli/broadcast_test.go

```diff
@@ -5,17 +5,20 @@ import (
 	"path/filepath"
 	"testing"
 
+	"github.com/cosmos/cosmos-sdk/client"
+	simappparams "github.com/cosmos/cosmos-sdk/simapp/params"
+
 	"github.com/spf13/viper"
 	"github.com/stretchr/testify/require"
 
 	"github.com/cosmos/cosmos-sdk/client/flags"
-	"github.com/cosmos/cosmos-sdk/codec"
 	"github.com/cosmos/cosmos-sdk/tests"
 )
 
 func TestGetBroadcastCommand_OfflineFlag(t *testing.T) {
-	cdc := codec.New()
-	cmd := GetBroadcastCommand(cdc)
+	clientCtx := client.Context{}
+	clientCtx = clientCtx.WithTxGenerator(simappparams.MakeEncodingConfig().TxGenerator)
```

Comment:
I better move this to a test helper.

Reply:
Can you track this to address in a separate PR @anilCSE ?